cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -190,6 +193,16 @@ public static String serviceName(String cluster) {
         return cluster + KafkaCluster.SERVICE_NAME_SUFFIX;
     }
 
+    /**
+     * Generates the name of the service used as bootstrap service for external clients
+     *
+     * @param cluster Name of the cluster
```

Comment:
Too many times "cluster"? :)

Reply:
Why? Its parameter named `cluster` which is the _Name of the cluster_. I think its ok that way.

Reply:
You are right :)